pkg/evaluators/identity.go

```diff
@@ -199,11 +199,13 @@ func (config *IdentityConfig) ResolveExtendedProperties(pipeline auth.AuthPipeli
 	authJSON := pipeline.GetAuthorizationJSON()
 
 	for _, extendedProperty := range config.ExtendedProperties {
-		resolved, err := extendedProperty.ResolveFor(extendedIdentityObject, authJSON)
-		if err != nil {
-			return nil, err
+		if extendedProperty.Value != nil {
```

Comment:
This took me forever to hunt down... Looks like there's always been uninitialized properties in there:

```
{"Name":"","Value":{"Static":null,"Pattern":""},"Overwrite":false},{"Name":"","Value":{"Static":null,"Pattern":""},"Overwrite":false}```
```

Which now have `"Value":nil` of course... So... I know I don't know. I _think_ this has been a bug in the making all along, but I don't know for sure... Someone with real deep knowledge and history of this might be able to make a call. I guess this is due to how the "slice" (which is effectively an array then... but I am no Go expert) is constructed [here](https://github.com/Kuadrant/authorino/pull/495/commits/a64198b704cecf80f6b1217c5a4aafadf5099ee0#diff-75405a152218cb714e868d6fe4cc86674992e73c8c0e0a9bd83fd3c475c4b7e0R184)